components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/workspace/workspace.js

```diff
@@ -313,11 +313,30 @@ define(['ace/ace', 'jquery', 'lodash', 'log','dialogs','./service-client','welco
                         saveMenuItem.disable();
                         saveAsMenuItem.enable();
                     }
-                    deletesMenuItem.enable();
                 } else {
                     saveMenuItem.disable();
                     saveAsMenuItem.disable();
-                    deletesMenuItem.disable();
+                }
+            };
+
+            this.updateDeleteMenuItem = function(){
+                var activeTab = app.tabController.getActiveTab(),
+                    deleteMenuItem = app.menuBar.getMenuItemByID('file.delete'),
+                    file = undefined;
+
+                if(activeTab.getTitle() != "welcome-page"){
```

Comment:
single quotes?

Reply:
Fixed in 01a6ed2d65f1fdec1b6be164e9550899def3a82b